src/components/auth/RegisterChildren.vue

```diff
@@ -651,10 +651,9 @@ const validateCode = async (studentCode, outerIndex = 0) => {
     console.error('Failed to validate activation code', error);
 
     if (!state.students[outerIndex].noActivationCode || props.code) {
-      dialogMessage.value = `The code ${studentCode} does not belong to any organization \n please enter a valid code or select: "I do not have a code"`;
+      dialogMessage.value = `The code ${studentCode} does not belong to any organization \n please enter a valid code`;
```

Comment:
The newline character is of no use in HTML. Instead I'd recommend simple punctuation: 

```suggestion
      dialogMessage.value = `The code ${studentCode} does not belong to any organization. Please enter a valid code.`;
```